docs/RELEASE-NOTES.rst

```diff
@@ -7,6 +7,19 @@ Release Notes for BIG-IP Controller for Kubernetes
 Added Functionality
 ```````````````````
 
+* Create health monitors for OpenShift Routes via an annotation.
+* Ability to disable loading of certificates and keys from Routes in preference of using pre-existing
+  profiles on the BIG-IP system.
+* Ability to disable loading Kubernetes Secrets on an Ingress.
```

Comment:
To keep consistent with voice of the existing contents, I would rephrase along the lines of:
"Optionally disable loading of ..."